models.js

```diff
@@ -94,4 +94,15 @@ function updateVote(newVote) {
         return result.rows[0].votes
     })
 }
-module.exports={getTopics, getAllEndPoints, getArtcileById, getArticleAndSort, getCommentsForArticle, addCommentForArticle, updateVote}
\ No newline at end of file
+
+function deleteComment(comment_id){
+    if(isNaN(comment_id)){
+        return Promise.reject({status: 400, message: 'Comment ID must be a number'})
+    }
+    return db.query(
+        `DELETE FROM comments WHERE comment_id = $1`, [comment_id]
+    ).then((result) =>{
+        return result.rows
```

Comment:
a hint for implementing a 404, if rows is empty here you could safely reject this promise